src/main/java/com/hartwig/snpcheck/PerlVcfComparison.java

```diff
@@ -13,7 +13,7 @@
 public class PerlVcfComparison implements VcfComparison {
 
     @Override
-    public Result compare(final Run run, final Blob refVcf, final Blob valVcf) {
+    public Result compare(final Run run, final Blob refVcf, final Blob valVcf, final Boolean alwaysPass) {
```

Comment:
I prefer to stick with the primitive `boolean` wherever possible as it avoids the possibility of `null`, which avoids surprises. Not a big deal in this instance but maybe something to keep in mind.